tests/integration-tests/tests/createami/test_createami.py

```diff
@@ -136,9 +147,22 @@ def test_build_image(
     _test_list_images(image)
     _test_export_logs(s3_bucket_factory, image, region)
 
-    lamda_vpc_config = image.config["DeploymentSettings"]["LambdaFunctionsVpcConfig"]
-    assert_lambda_vpc_settings_are_correct(
-        image.image_id, region, lamda_vpc_config["SecurityGroupIds"], lamda_vpc_config["SubnetIds"]
+
+@retry(
+    retry_on_result=lambda result: result == "CREATE_IN_PROGRESS",
+    wait_fixed=2000,  # 2 seconds
```

Comment:
Two seconds might be too short and causing API rate exceeded. Moreover, we have a utils function `seconds`
So this line can be:
```
 wait_fixed=seconds(10)
```

Moreover, can you also add a timeout:
```
stop_max_delay=seconds(200)
```